platform/engineering/code_review_guidelines.md

```diff
@@ -4,41 +4,58 @@ This document covers rules for engaging the VSP team in [pull request reviews](h
 
 * Help maintain the code quality and stability of the platform itself
 * Enable VSP team to quickly review changes for VFS teams
+* Clarify the process for moving code from development through review to production
 
-Before sending your code to a VSP team for review, your code should meet all of the guidelines below.
-
-*Hint*: smaller, more frequent pull requests will make this easier for everyone!
-
-## Guidelines
-
-* **Review lifecycle**
-    * Before sending code to VSP team have it reviewed by your team (see **First review** below) using a [Draft Pull Request](https://github.blog/2019-02-14-introducing-draft-pull-requests/).  
-      * This prevents [code owners](https://help.github.com/en/articles/about-code-owners) from being notified before pull request is ready for review.
-    * After having your pull request reviewed by your team you can click the *Ready for review* button in the merge section of your pull request.
-      * An engineer from the VSP team will aim to review within 24 hours (business days) after the pull request has been reviewed by your team and you have been marked it *Ready for review*. Each additional round of changes will be reviewed within 24 hours.  If you need a review faster, escalate via your DSVA product owner.
```

Comment:
this update also loses the clarity about "additional rounds of changes", which IMO we should keep explicitly called out

Reply:
Thanks! That literally just got dropped in edit.